modules/orca/main.tf

```diff
@@ -122,6 +122,8 @@ module "orca_secretsmanager" {
   db_user_password  = var.db_user_password
   db_host_endpoint  = var.db_host_endpoint
   restore_object_role_arn = module.orca_iam.restore_object_role_arn
+  s3_access_key = ""
```

Comment:
I do not yet know where/how this value will be pulled in.Seems possible that this will be generated within the orca module subsidiaries.

Reply:
Since the only way to generate these currently to meet NASA guidelines is via Cloudtamer, an end user will need to pass these is via variables either through `terraform -e` or a `terrform.tfvars` file. So variables will be needed for both of these values.

Reply:
I understand that we are not allowed to create a user within TF. Here is an alternate method for generating keys for a given iam user. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_access_key